packages/maskbook/src/components/InjectedComponents/AdditionalPostContent.tsx

```diff
@@ -66,11 +66,37 @@ export const AdditionalContent = memo(function AdditionalContent(props: Addition
             {props.headerActions}
         </Typography>
     )
+
+    const imgData = useMemo(() => {
+        if (message && typeof message !== 'string') {
+            const s = extractTextFromTypedMessage(message).val
```

Comment:
Hmm you should use `TypedMessageImage` to represent the image instead of using `b64Image=` image

Reply:
i wasn't aware of its existence, thank you. apologies for all of the necessary revisions, still getting acquainted with the code base